bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -56,6 +57,15 @@ pub enum Command {
 		/// XCM execution fee on AssetHub
 		fee: u128,
 	},
+	/// Send Polkadot token back to the original parachain
+	SendForeignToken {
```

Comment:
This would be a Native token? We are receiving a Polkadot native token send command here on the inbound queue.

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/128/commits/cf091f69547cced2473151f70ebf2013f2a14591